index.html

```diff
@@ -422,7 +425,6 @@
             margin: 0;
             overflow: hidden;
```

Comment:
We can also remove `overflow: hidden` and `text-overflow: ellipsis` now that there no longer is any overflow